packages/framework/tests/Unit/Component/HttpFoundation/FragmentHandlerTest.php

```diff
@@ -15,7 +15,7 @@ class FragmentHandlerTest extends TestCase
     public function testRenderNotIgnoreErrorsWithoutDebug()
     {
         $rendererMock = $this->createMock(FragmentRendererInterface::class);
-        $rendererMock->expects($this->once())->method('getName')->willReturn('rendererName');
+        $rendererMock->expects($this->any())->method('getName')->willReturn('rendererName');
```

Comment:
what is the reason for this change? and others similar

Reply:
Because `getName` method has been called in multiple places in symfony4 (in this case, the renderer is stored in the key field that `getName` returns and `addRenderer` is called twice, at first in test, and at second in `LazyLoadingFragmentHandler:render`)